### PR TITLE
propose 2024-02-28 meeting

### PR DESCRIPTION
**Pull request by chrisdickinson**
Looks like a pretty light agenda this time around. We may be welcoming several new members (:tada:) so we might walk through the demo-driven approach we proposed in the last meeting.

### CODE REVIEW

**Review by brooksmtownsend — APPROVED** [review]

File: meetings/2024-02-28.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+### Action items
+
+- New folks -- please go read the user stories on the readme and open issues if you find anything
+  that strikes you as incorrect!
+  - Anyone who is interested, please try implementing [roadmap demo 0](https://github.com/WebAssembly/wasi-observe/blob/main/ROADMAP.md#demo-0)
+    - multiple implementations are encouraged!
+- Everyone -- we're going to try out github discussions for our communication channel
+- Review the [WIT draft PR](https://github.com/WebAssembly/wasi-observe/pull/12/files)
+
+[brooksmtownsend]: https://github.com/brooksmtownsend
```
Comment (calebschoepp) [inline meetings/2024-02-28.md:63]: Nit: Double brooks 😄

**Review by calebschoepp — APPROVED** [review]
LGTM

File: meetings/2024-02-28.md (hunk 1 above)
Comment (brooksmtownsend) [reply meetings/2024-02-28.md:63]: I for one welcome my companion

Comment (calebschoepp) [reply meetings/2024-02-28.md:63]: More the merrier

**Pull request merged**